azure-pipelines/integrationtests.yml

```diff
@@ -67,13 +67,13 @@ steps:
   - task: DotNetCoreCLI@2
     displayName: Building Integration Tests
     inputs:
-      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj'
+      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj, **/Microsoft.SqlTools.SqlCore.IntegrationTests.csproj'
 
   - task: DotNetCoreCLI@2
     displayName: Running Integration Tests
     inputs:
       command: test
-      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj'
+      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj, **/Microsoft.SqlTools.SqlCore.IntegrationTests.csproj'
       arguments: '--no-build'
       testRunTitle: 'SqlToolsService Integration Tests'

```

Comment:
nit: I'd probably suggest making a separate task for the SqlCore integration tests so that they're grouped up correctly in the UI